src/platform/julius.c

```diff
@@ -311,14 +307,13 @@ static void main_loop(void)
     int quit = 0;
     while (!quit) {
         SDL_Event event;
+#ifdef PLATFORM_ENABLE_PER_FRAME_CALLBACK
+        platform_per_frame_callback();
+#endif
         /* Process event queue */
 #ifdef __vita__
-        vita_handle_analog_sticks();
-        vita_handle_virtual_keyboard();
         while (vita_poll_event(&event)) {
 #elif defined(__SWITCH__)
-        switch_handle_analog_sticks();
-        switch_handle_virtual_keyboard();
         while (switch_poll_event(&event)) {
 #else
         while (SDL_PollEvent(&event)) {
```

Comment:
Can we also `#define` a poll event function so we don't have the preprocessor switch here?

Reply:
My plan is to completely get rid of platform-specific poll events, that's why I didn't do it here :wink:

If you want I can do as you requested, but it'll be a temporary change.

Reply:
Then it's ok as-is, no need for a temporary change.